cmd/pod.go

```diff
@@ -67,6 +75,17 @@ func (p *podHandler) OnDelete(obj interface{}) {
 	}
 }
 
+func keysDiffer(oldAnnotations, newAnnotations map[string]string, keysThatMatter []string) bool {
```

Comment:
1. Why using an array of string as we'll always pass only one key?
1. It's not the key that differs IIUC, it's the annotation value so I would rename the function to `annotationDiffers`